tiledb/sm/serialization/rest_version.h

```diff
@@ -0,0 +1,71 @@
+/**
+ * @file   rest_version.h
+ *
+ * @section LICENSE
+ *
+ * The MIT License
+ *
+ * @copyright Copyright (c) 2025 TileDB, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ * @section DESCRIPTION
+ *
+ * This file declares serialization functions for REST version information.
+ */
+
+#ifndef TILEDB_SERIALIZATION_REST_VERSION_H
+#define TILEDB_SERIALIZATION_REST_VERSION_H
+
+#ifdef TILEDB_SERIALIZATION
+#include "tiledb/sm/serialization/capnp_utils.h"
+#endif
+
+using namespace tiledb::common;
+
+namespace tiledb::sm {
+
+class RestCapabilities;
+class SerializationBuffer;
+enum class SerializationType : uint8_t;
+
+namespace serialization {
+
+#ifdef TILEDB_SERIALIZATION
+
+RestCapabilities rest_version_deserialize(
+    SerializationType serialization_type, span<const char> serialized_response);
+
+RestCapabilities rest_version_from_capnp(
+    const capnp::RestVersion::Reader& rest_version_reader);
+
+#else
+
+RestCapabilities rest_version_deserialize(
+    SerializationType serialization_type, span<const char> serialized_response);
+
+RestCapabilities rest_version_from_capnp(
+    const capnp::RestVersion::Reader& rest_version_reader);
+
+#endif  // TILEDB_SERIALIZATION
```

Comment:
```suggestion
RestCapabilities rest_version_deserialize(
    SerializationType serialization_type, span<const char> serialized_response);

RestCapabilities rest_version_from_capnp(
    const capnp::RestVersion::Reader& rest_version_reader);
```

Aren't these declarations identical?

Reply:
Yep, I yanked them from the source file in vim and forgot to delete the preprocessor checks